types/app.d.ts

```diff
@@ -24,9 +24,8 @@
 
 declare namespace Realm {
     // See https://stackoverflow.com/a/51114250 on why we're importing the BSON types like this
-    type ObjectId = import("bson").ObjectId;
```

Comment:
Any reason this is getting removed? It's used by `services.d.ts` and `auth-providers.d.ts`.

Reply:
We might want to move it to line 27 of index.d.ts if we want to use it there as well.